pages/project/revoke/index.js

```diff
@@ -0,0 +1,33 @@
+const { page } = require('@asl/service/ui');
+const confirm = require('./routers/confirm');
+const success = require('../../common/routers/success');
+const revoke = require('./routers/revoke');
+
+module.exports = () => {
+  const app = page({
+    root: __dirname,
+    paths: ['/confirm', '/success']
+  });
+
+  app.use((req, res, next) => {
+    req.breadcrumb('project.revoke');
+    req.model = req.project;
```

Comment:
could we use an empty object here? the `model` in a form only relates to the form values, it also uses the model id for session storage and cleanup - I'd go with

```
req.model = {
  id: `${req.project.id}-revoke`
};
```